app/utils/track.js

```diff
@@ -203,6 +203,7 @@ export function trackPageView(isInitialPageView, uaDimensions) {
     M.trackingQueue.push(window.trackQuantcastPageView);
     M.trackingQueue.push(window.trackComscorePageView);
     M.trackingQueue.push((isOptedIn) => {
+      window.fandomFCPtrackPageView(false);
```

Comment:
`false` to prevent increasing the page view number as it will be increased by regular page view event